doc/manual/src/language/derivations.md

```diff
@@ -8,98 +8,204 @@ It outputs an attribute set, and produces a [store derivation] as a side effect
 
 [store derivation]: @docroot@/glossary.md#gloss-store-derivation
 
-<!-- FIXME: add a section on output attributes -->
-
 ## Input attributes
 
 ### Required
 
 - [`name`]{#attr-name} ([String](@docroot@/language/values.md#type-string))
 
   A symbolic name for the derivation.
-  It is added to the [store derivation]'s [path](@docroot@/glossary.md#gloss-store-path) and its [output paths][output path].
-
-  Example: `name = "hello";`
+  It is added to the [store path] of the corresponding [store derivation] as well as to its [output paths](@docroot@/glossary.md#gloss-output-path).
+
+  [store path]: @docroot@/glossary.md#gloss-store-path
+
+  > **Example**
+  >
+  > ```nix
+  > derivation {
+  >   name = "hello";
+  >   # ...
+  > }
+  > ```
+  >
+  > The store derivation's path will be `/nix/store/<hash>-hello.drv`.
+  > The [output](#attr-outputs) paths will be of the form `/nix/store/<hash>-hello[-<output>]`
```

Comment:
Do you want to mention how `out` is special?

Reply:
It is mentioned in the outputs section.

Reply:
Oh is the `[...]` supposed to indicate that suffix may or may not appear?